src/winfile.exe.manifest

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+	<assemblyIdentity
+		version="1.0.0.0"
+		processorArchitecture="*"
+		name="winfile.exe"
+		type="Win32"
+		/>
```

Comment:
We don't appear to need the "assemblyIdentity" section, do we?  Such a section introduces another version number to keep up to date.